.buildkite/scripts/common.sh

```diff
@@ -503,6 +503,10 @@ prepare_stack() {
         fi
     fi
 
+    if [ "${STACK_LOGSDB_ENABLED}" == "true" ]; then
```

Comment:
It could be added here the default value:

```suggestion
    if [ "${STACK_LOGSDB_ENABLED:-false}" == "true" ]; then
```

or add to the pipeline definition something like:

```
env:
  STACK_LOGSDB_ENABLED: "${STACK_LOGSDB_ENABLED:-false}"
```